apps/api/src/providers/secrets/index.ts

```diff
@@ -0,0 +1,39 @@
+import { EmailProviderIdEnum } from '@novu/shared';
+import axios from 'axios';
+
+const getDopplerSecrets = async () => {
+  const response = await axios.get(
+    `https://${process.env.DOPPLER_TOKEN}@api.doppler.com/v3/configs/config/secrets/download?format=json`
+  );
+
+  return response.data;
+};
```

Comment:
This returns an object with the project and secrets related to the doppler token provided. As you can see there is no mention of project names and so and it is good as a safety measure. But at the very same time one has to be careful with the token provided to not inject non desired secrets.